aframe/pipelines/sandbox/configs/bns.cfg

```diff
@@ -27,6 +27,7 @@ waveform_approximant = IMRPhenomPv2
 waveform_duration = 90
 minimum_frequency = 20
 reference_frequency = 50
+coalescence_time = 45
```

Comment:
Does it make sense to define the placement time w.r.t to the right edge of the kernel (i.e. it would be negative)?

Reply:
To me, it's more intuitive to make this (as well as the left and right pads) defined to be positive, so that there's no chance of hidden negatives in adding and subtracting distances. And given that I wanted it to be more positive, I thought that the term "coalescence time" made more sense when defined with respect to the left edge of the kernel.